lua/mason-registry/curlylint/init.lua

```diff
@@ -0,0 +1,11 @@
+local Pkg = require "mason-core.package"
+local pip3 = require "mason-core.managers.pip3"
+
+return Pkg.new {
+    name = "curlylint",
+    desc = [[Experimental HTML templates linting for Jinja, Nunjucks, Django templates, Twig, Liquid]],
+    homepage = "https://www.curlylint.org/",
+    languages = { Pkg.Lang.Python },
```

Comment:
For some tools the languages to include can be a bit awkward, but I think the following would be most correct:

```suggestion
    languages = { Pkg.Lang.Django, Pkg.Lang.Ninja, Pkg.Lang.Nunjucks, Pkg.Lang.Twig, Pkg.Lang.Liquid },
```